nodestream/cli/commands/shared_options.py

```diff
@@ -19,3 +19,10 @@
 MANY_PIPELINES_ARGUMENT = argument(
     "pipelines", "the names of the pipelines", multiple=True, optional=True
 )
+
+DEVELOPMENT_ENVIRONMENT_OPTION = option(
+    "development",
+    "g",
+    "Run development settings. Cronjobs are suspended, perpetuals are scaled to 0 unless the debug annotation is specified.",
+    flag=True,
+)
```

Comment:
There seems to be no reason why this was added to nodestream core aside from its use in https://github.com/nodestream-proj/nodestream-plugin-k8s/pull/6/files. Is that right?

If so, I think it would be better to move this to that PR.